src/App.js

```diff
@@ -1,11 +1,12 @@
 import React from "react";
 import "./App.css";
+import Score from "./Score"
 
 class App extends React.Component {
   render() {
     return (
       <main>
-        <p>React State Lab</p>
+        <Score />
```

Comment:
Since you're only rendering one component and nothing else here, you **could** just take everything from Score.js and just put the whole thing in App.js.